docs/final report/chapters/software-overview.tex

```diff
@@ -1,12 +1,29 @@
 \chapter{Software Overview} %MAX 1 PAGE
-
-Introduction to software overview. (what has been made, all must haves implemented, etc). 1 PAGE
+At the start of the project we constructed a product planning in which we described what features we want to implement. In this chapter we will describe to what extend these features are truly implemented.
 
 \section{Must Have}
-Describe all must haves that have been implemented.
+There were features that were obligatory to have in the software for it to be able to work properly. The next items are the must-haves that we have implemented in our program.
+\begin{description}
+
+\item[Language] We implemented the declarative language SUGAR and can be used to perform analyses. 
+
+\item[Analysis] The program can perform chunk, constraint, comparison, computation, code and connection operations on data. All these can be edited and performed in a simple editor. It is also possible to load an existing script from a text file which can be executed by the program.
+
+\item[Load Data] We can specify measurement data in an xml file. The files can then be loaded into the program so that analyses can be performed.  We also implemented a graphical wizard to create a specification of datafiles which can then be saved to an xml file.
+
+\item[Visualization] We have implemented features that the user can create barchart, boxplot and state-transition matrix visualizations. It is possible to create a frequency bars plot  by making a counting analysis and plotting it into a barchart.
+
+\item[datatypes] It is possible to distinguish different datatypes. We have implemented classes to describe integers, floats, booleans, strings, dates, times and datetimes. The types can be distinguished in the specification of the datafile and these types will then be used to perform the analyses on.
```

Comment:
Dit is geen must have requirement. Dit is misschien nodig voor het pogramma, maar het is geen user requirement.